.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+// Place your settings in this file to overwrite default and user settings.
```

Comment:
Was this comment intentionally left in, or written?  The consumer of the module would not be using this file, right?